metrotranscom/doorway
# Refactor/normalize etl task names

# Pull Request Template

## Issue Overview

This PR addresses (N/A)


## Description

Some of the naming used for the listings import task was inconsistent with previously established naming conventions and/or was not sufficiently descriptive.  This PR changes the directory that the task is in and renames a var to more closely align with comparable vars elsewhere in the repo, and adds comments noting relationships with other vars.

## How Can This Be Tested/Reviewed?

This PR does not change any code and does not require retesting.  If you want to verify that the task works as expected, you can run `yarn test` or `yarn import:run` (after setting env vars).

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
